launcher/game/updater.rpy

```diff
@@ -178,6 +218,20 @@ screen updater:
 label update:
 
     python hide:
+        criteria = ""
```

Comment:
Think this can be moved to a separate label to just go direct to the list then search it as a option